packages/validation/src/references/utils/getResourceNodes.ts

```diff
@@ -6,6 +6,7 @@ import {
   ResourceRefsProcessingConfig,
 } from "../../common/types.js";
 import { REF_PATH_SEPARATOR, NAME_REFNODE_PATH } from "../../constants.js";
+import isDeepEqual from 'fast-deep-equal/es6/react';
```

Comment:
Any reason why you use the React version? Seems a bit out of place as this is unrelated to frontend components.

Reply:
ok - updated!